packages/fxa-settings/package.json

```diff
@@ -33,21 +33,25 @@
   },
   "dependencies": {
     "@apollo/client": "^3.3.7",
+    "@emotion/react": "^11.1.4",
```

Comment:
???

Reply:
`@emotion/react` is supposed to be a devDependency for `@material-ui/core/slider`. But I wasn't able to get it working without installing it also. I'm not sure if there is another work around for this.